.github/workflows/pythonpackage.yml

```diff
@@ -20,15 +20,19 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install -r requirements.txt
+        curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py > get-poetry.py
+        python get-poetry.py --version 1.0.2 -y
+        $HOME/.poetry/bin/poetry install
+        rm get-poetry.py
     - name: Lint with flake8
       run: |
-        pip install flake8
+        # activate the virtualenv created by poetry
+        source .venv/bin/activate
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # the GitHub editor is 127 chars wide
         flake8 . --count --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest
       run: |
-        pip install pytest
+        source .venv/bin/activate
```

Comment:
Necessary change for this PR?

Reply:
Same as above, pytest is included in the dev requirements and for some reason the venv doesn't stay activated between steps so I have to activate it again.